xml/systemd.xml

```diff
@@ -1485,51 +1485,75 @@ UMask=007</screen>
     <sect2 xml:id="sec-boot-systemd-custom-drop-in">
       <title>Creating drop-in files</title>
       <para>
-        For minor changes of a configuration file, use so-called drop-in files.
-        Drop-in files let you extend the configuration of unit files without
-        having to edit or override the unit files themselves.
+        For minor changes of a configuration file, use so-called drop-in files
+        or <emphasis>drop-ins</emphasis>. Drop-ins let you extend the
+        configuration of unit files without having to edit or override the unit
+        files themselves. Drop-ins have higher precedence over main
+        configuration files. They are stored in the following directories:
```

Comment:
There are two classes of config files: _unit_ files and daemon config files.
So drop-ins let you extend both classes of configs.